src/containers/About/Issues.js

```diff
@@ -29,22 +30,28 @@ class Issues extends React.Component {
       sort: "created",
       labels: "",
       assignee: ""
-    }
+    },
+    initialized: false,
+    loadingSignal: false
   };
 
   componentWillMount() {
-    this.props.readContributors();
+    this.props.readContributors()
+      .then(() => this.setState({initialized: true}));
   };
 
   updateFilter = (key, value) => {
+    this.setState({loadingSignal: true});
     const newFilter = Object.assign(this.state.filter, { [key] : value });
     if(!value) delete newFilter[key];
     this.setState({ filter: newFilter });
-    this.props.readIssues(newFilter);
+    this.props.readIssues(newFilter)
+      .then(() => this.setState({loadingSignal: false}));
   };
 
   render() {
-    if(!this.props.contributors.length) return (<div>LOADING</div>);
+    const loading = (<center> <CircularProgress /> </center> );
```

Comment:
`<center>` tag has been deprecated from HTML
http://stackoverflow.com/questions/11478067/replacement-for-center